guides/release/components/conditional-content.md

```diff
@@ -72,6 +72,17 @@ _public API_ of the component, so an argument makes sense.
 So, we want to add the `is-active` class if an argument, like say `@isActive`,
 is passed in and is truthy.
 
+```handlebars {data-filename="app/components/avatar.hbs"}
```

Comment:
I moved this up so the reader could go straight to the example before we explain our falsy approach in templates. It was jarring otherwise